rllib/utils/pre_checks/env.py

```diff
@@ -1,9 +1,13 @@
 """Common pre-checks for all RLlib experiments."""
 import logging
+from typing import TYPE_CHECKING
 
 import gym
 from ray.rllib.utils.typing import EnvType
 
+if TYPE_CHECKING:
+    from ray.rllib.env import MultiAgentEnv
```

Comment:
can you tell me why rllib/env/multi_agent_env.py needs to import pre_checks/env.py?
or if you have a command to repro this, I can take a look too.